content/post/094-gossipsub-v1.1.md

```diff
@@ -0,0 +1,79 @@
+---
+date: 2020-05-19
+url: 2020-05-19-gossipsub-v1.1
+title: Gossipsub v1.1 brings hardening extensions to PubSub 
+author: David Dias
+header_image: 094-gossipsub-v1.1-headerimage.jpeg
+tags: ipfs, libp2p, pubsub, gossipsub
+---
+
+<p align="center">
+  <img width="60%" src="/header_images/094-gossipsub-v1.1-headerimage+grid.jpeg">
+</p>
+
+The gossip you’ve heard on the streets is correct, Gossipsub v1.1 is here and it packs several security hardening extensions - along with new testing, documentation, and an updated specification.
+
+The Gossipsub Task Force has been hard at work exploring and analyzing various attack vectors on public and permissionless messaging networks. With that knowledge, we’ve crafted and iterated on mitigating strategies to make our beloved libp2p PubSub Router work in adversarial environments. The result is Gossipsub v1.1 🚀
+
+If you are new to libp2p PubSub and especially Gossipsub, we recommend you checking out the [**Gossipsub v1.1 talk at the Matrix Virtual Meetup**](https://research.protocol.ai/blog/2020/gossipsub-v1.1-at-open-tech-will-save-us-virtual-event). It will give you the complete motivation and background of libp2p PubSub and introduce you to Gossipsub v1.1.
+
+## 🔍 What is Gossipsub v1.1
+
+The main focus for Gossipsub v1.1 is **security**. In addition to the speedy message propagation guarantees from v1.0, this new version features several hardening extensions that make Gossipsub more resilient to a wide range of attacks. Some highlights include:
+
+**1) [Peer scoring](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#peer-scoring)**: peers now monitor their directly connected neighbours using a score function that reflects the peer’s usefulness. Well-behaving peers are kept in the mesh, while badly-behaving ones are dropped in order to protect the network from malicious actors.
+
+**2) [Adaptive gossip dissemination](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#adaptive-gossip-dissemination)**: In this new version, you can adjust the `gossip factor` to control the number of peers your node gossips with. This enables you to increase/decrease the amount of gossip, while keeping a baseline.
+
+**3) [Opportunistic Grafting](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#opportunistic-grafting)**: Grafting is a new mechanism used to cherry pick well-behaving peers that are currently not part of a node’s mesh. Those well-behaving peers are then opportunistically inserted into the mesh to improve the median score of participating peers. 
+
+**4) [Prune Peer eXchange](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#prune-backoff-and-peer-exchange)**: Prune Peer exchange gives a node a set of recommended peers to connect when it gets dropped from another peer’s mesh, helping reduce the dependency on ambient peer discovery mechanisms.
```

Comment:
we also have to mention the backoff I think

Reply:
Folks can always read the spec. For this blog post, our goal is to have something that works as a highlights/gist, so that folks can feel they grok it but also know where to look for more details (i.e. the spec) when they decide to.